featuretools/computational_backends/calculate_feature_matrix.py

```diff
@@ -283,61 +283,61 @@ def calculate_feature_matrix(features, entityset=None, cutoff_time=None, instanc
 
     progress_bar = make_tqdm_iterator(**tqdm_options)
     progress_bar._instances.clear()
```

Comment:
do we need this progress bar instance?

Reply:
nope, that was accidentally left in with a merge